lms/static/sass/course/modules/_calculator.scss

```diff
@@ -82,6 +94,13 @@ div.calc-main {
         &:hover, &:focus {
           color: #333;
         }
+
+        span.sr {
```

Comment:
Hmm, any chance we can give this button another class (like ``calculator_button-label``) so we can keep the rules scoped, but tied to a class rather than an element?